build/media_source/com_scheduler/css/admin-view-select-task.css

```diff
@@ -0,0 +1,58 @@
+.new-task {
```

Comment:
It seems, com_modules has no CSS for their select, so we should avoid this, too.

Reply:
I believe com_modules has their CSS in the template file, should we go with that and integrate this into Cassiopeia?